docs: docs about search box in download snapshot (#5628)
eclipse-kura/kura: docs/administration/snapshot-management.md

Starting with Kura 6.0, it is possible to download _a partial_ version of the snapshot. This allows the user to download a subset of the entire Kura configuration.

Pressing the **Download** button will trigger a dialog that allows choosing the desired configurations and format.

![](images/snapshotsDownload.png)

> In the snapshot download popup, a list of checkboxes is shown, representing the list of all the service configurations that can be downloaded. By default, all configurations (represented by the component PIDs) are selected. The user can remove from the downloaded snapshot the configuration they're not interested in by deselecting the related PIDs checkboxes. The snapshot can be then downloaded in XML or JSON format thorough the dedicated button.

Reviewer: ```suggestion
In the snapshot download popup, a list of checkboxes is shown, representing the list of all the service configurations that can be downloaded. By default, all configurations (represented by the component PIDs) are selected. The user can remove from the selection the configurations he's not interested in, by deselecting the related PIDs checkboxes. Only the selected component configurations will be then downloaded.

The snapshots can be downloaded in XML or JSON format.
```

In the snapshot download popup, a list of checkboxes is shown, representing the list of all the service configurations that can be downloaded. By default, all configurations (represented by the component PIDs) are selected. The user can remove from the downloaded snapshot the configuration they're not interested in by deselecting the related PIDs checkboxes. The snapshot can be then downloaded in XML or JSON format thorough the dedicated button.

> Through the search box above the list, it is possible to easily find the desired services configuration, filtering them by their PIDs.

Reviewer: A search box is available for the user to simplify the process of PID selection for the configurations available in the selected snapshot. As an example, in the screenshot below, it can be seen the filtering in progress as the user writes its filtering keyword

Pressing the **Download** button will trigger a dialog that allows choosing the desired configurations and format.

![](images/snapshotsDownload.png)

> ![](images/snapshotsDownloadFiltered.png)

Reviewer: This can be moved up to connect with the specific sentence

Reviewer: be careful that the screenshots are not up to date
